src/lemlib/odom/differentialArc.cpp

```diff
@@ -144,33 +144,36 @@ void DifferentialArc::update() {
         infoSink()->error("Odom calculation failure! Not enough sensors to calculate heading");
         return;
     }
-    const float deltaTheta = theta - pose.theta;
-    const float avgTheta = pose.theta + deltaTheta / 2;
+    const Angle deltaTheta = theta - pose.theta;
+    const Angle avgTheta = pose.theta + deltaTheta / 2;
 
     // calculate local change in position
-    Pose local(0, 0, deltaTheta);
+    Pose local(0_in, 0_m, deltaTheta);
```

Comment:
```suggestion
    Pose local(0_m, 0_m, deltaTheta);
```
Internal units should be consistent